test/positive_numbers.jl

```diff
@@ -0,0 +1,45 @@
+include("utils.jl")
+
+@testset "Positive Numbers" begin
+    M = PositiveNumbers()
+    @testset "Positive Numbers Basics" begin
+        @test repr(M) == "PositiveNumbers()"
+        @test repr(PositiveVectors(2)) == "PositiveVectors(2)"
+        @test repr(PositiveMatrices(2, 3)) == "PositiveMatrices(2, 3)"
+        @test repr(PositiveArrays(2, 3, 4)) == "PositiveArrays(2, 3, 4)"
+        @test representation_size(M) == ()
+        @test manifold_dimension(M) == 1
+        @test !is_manifold_point(M, -1.0)
+        @test_throws DomainError is_manifold_point(M, -1.0, true)
+        @test is_tangent_vector(M, 1.0, 0.0; check_base_point = false)
+        @test flat(M, 1.0, FVector(TangentSpace, 1.0)) == FVector(CotangentSpace, 1.0)
+        @test sharp(M, 1.0, FVector(CotangentSpace, 1.0)) == FVector(TangentSpace, 1.0)
+        @test vector_transport_to(M, 1.0, 3.0, 2.0, ParallelTransport()) == 6.0
+        @test retract(M, 1.0, 1.0) == exp(M, 1.0, 1.0)
+        @test isinf(injectivity_radius(M))
+        @test isinf(injectivity_radius(M, -2.0))
+        @test isinf(injectivity_radius(M, -2.0, ExponentialRetraction()))
+        @test isinf(injectivity_radius(M, ExponentialRetraction()))
+        @test project(M, 1.5, 1.0) == 1.0
+        @test zero_tangent_vector(M, 1.0) == 0.0
+    end
+    types = [Float64]
+    TEST_FLOAT32 && push!(types, Float32)
+    for T in types
+        @testset "Type $T" begin
+            pts = convert.(Ref(T), [1.0, 4.0, 2.0])
+            test_manifold(
```

Comment:
I think you should also call `test_manifold` on the `PositiveVectors` manifold.

Reply:
No, it's 3 points on `PositiveNumbers()` not one point on the `PositiveVectors(3)`.

Reply:
Yes, that's right, what I had in mind was adding another call to `test_manifold` below that tests the manifold `PositiveVectors(3)`.

Reply:
Just to make sure nothing breaks there.

Reply:
It's merely an alias for a PowerManifold, so that should not be necessary,

Reply:
I know but it's easy to write functions on `PositiveNumbers` in a way that's not compatible with `PowerManifold`. I would even say, it most likely won't work as is since it generally assumes that points are represented by array-like objects (including points on submanifolds). With Circle/Torus we actually allowed points on Circle to be represented by one-element arrays.

Reply:
Ok, you might be right, I'll do such a test tomorrow and maybe add also the array variants if necessary.

Reply:
had a minute - there it is.